src/db/helper.c

```diff
@@ -63,9 +63,9 @@ dbGet(bool primaryOnly, bool primaryRequired, bool standbyRequired)
     MEM_CONTEXT_TEMP_BEGIN()
     {
         // Loop through to look for primary and standby (if required)
-        for (unsigned int pgIdx = 0; pgIdx < cfgOptionIndexTotal(cfgOptPgPath); pgIdx++)
+        for (unsigned int pgIdx = 0; pgIdx < cfgOptionGroupIndexTotal(cfgOptGrpPg); pgIdx++)
         {
-            if (cfgOptionTest(cfgOptPgHost + pgIdx) || cfgOptionTest(cfgOptPgPath + pgIdx))
+            if (cfgOptionGroupIndexTest(cfgOptGrpPg, pgIdx))
```

Comment:
Again, since we're no longer testing to see if the options cfgOptPgHost, cfgOptPgPath are set, only that the group has at least one value set - do we need to check? Maybe the parser/config system already do that? If so, maybe you can point me to file/line where this is done?

Reply:
Same as above.